### PR TITLE
Clearify CLM environment status message

### PR DESCRIPTION
**Pull request by ncounter**
## What does this PR change?

On a CLM environment status the message says `Generating repositories data` which is a misleading sentence because it suggests the generation is happening right now behind the scenes. That is not really true: reality is that it happens in `async` by the `channel-repodata-default` task, and depending on the schedule value of the task it can take a while **before starting** the generation of such data. So the message is that we are rather _waiting for the data to be generated_

## GUI diff

The words are slightly different

- [x] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- No documentation needed: only internal and user invisible changes
- Documentation issue was created: [Link for SUSE Manager contributors](https://github.com/SUSE/spacewalk/issues/new?template=ISSUE_TEMPLATE_DOCUMENTATION.md&labels=documentation&projects=SUSE/spacewalk/31), [Link for community contributors](https://github.com/uyuni-project/uyuni-docs/issues/new).
- (OPTIONAL) [Documentation PR](https://github.com/uyuni-project/uyuni-docs/pulls)

- [x] **DONE**

## Test coverage
- No tests: no tests for this a string

- [x] **DONE**

## Links

Fixes # just a helper spin-off of https://github.com/SUSE/spacewalk/issues/15261
Tracks # **add downstream PR, if any**

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by Etheryte — APPROVED** [review]
Linguistically the sentence feels a bit clunky, but I'm not quite sure how to make it sound "native" while also remaining reasonably accurate. I take it that `"Waiting for repositories to be generated"` would be inaccurate since the repositories themselves already exist? `"Waiting for repositories to be populated"`? `"Waiting for repository data to be generated"` would be an option too, although again, perhaps not the most accurate.

In short, I don't know, I think we need to hire a linguist. 😄

From my side you can merge as-is as well since it's more accurate than what was there before.

**Comment by ncounter** [comment]
> 



> Linguistically the sentence feels a bit clunky, but I'm not quite sure how to make it sound "native" while also remaining reasonably accurate. I take it that `"Waiting for repositories to be generated"` would be inaccurate since the repositories themselves already exist? `"Waiting for repositories to be populated"`? `"Waiting for repository data to be generated"` would be an option too, although again, perhaps not the most accurate.
> 
> In short, I don't know, I think we need to hire a linguist. smile
> 
> From my side you can merge as-is as well since it's more accurate than what was there before.

Thank you for the quick review.

I agree and I know my English is not a very good one :man_facepalming: 

I will ask a review from our doc guy who is native (cc @jcayouette).

About the `populated` option, I think that would be partially wrong as well because the message is both for when the channels are freshly created and empty, but also for when they get rebuilt so repodata are re-generated but channels are already _populated_ . Anyway, I am fully open to any better wording, my only care is about the meaning of the timing that can be extracted from the words.

For the record, it'd be much better to mention about the async chain and the task that is taking care of it, but we should stay transparent in that regards as it would add a weak point to the chain if in the future the mechanism will change.

**Review by cbbayburt — APPROVED** [review]
Nice catch, thanks, @ncounter!

I'm not sure if we need a changelog entry though. Even though it's super trivial for the user, it's still a user-facing change and one might get curious when they see a different message.
